scripts/aspenmesh-collector-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+if ! [ -x "$(command -v helm)" ]; then
+  echo "helm is not installed. Please visit https://helm.sh/docs/intro/install/" >&2
+  exit 1
+fi
+
+usage() {
+  echo "
+usage: ${0} [flags] [-h|--help]
+Flags               Purpose
+--apiKey            Required; The FusionAuth API key for the user's group
```

Comment:
The user wouldn't know what FusionAuth is, so suggest re-phrasing as:
```suggestion
--apiKey            Required; The API key provided upon initial signup to Aspen Mesh
```